topic_info: {
  current_tesselation_topic: "/voronoi/tesselation",
  occupancy_grid_topic: "/map",
  occupancy_grid_service: "static_map",
  speed_topic: "cmd_vel",
  pose_topic: "base_pose_ground_truth",
  robot_prefix: "robot_",
  density_topic: "/voronoi/density_dist"
}